catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/history/HistorianTest.java

```diff
@@ -202,6 +203,9 @@ public void testUpdateStorageResponseSetSkipFlag()
     when(storageResponse.getProperties()).thenReturn(storageResponseProperties);
 
     UpdateResponse updateResponse = mock(UpdateResponse.class);
+    Update update1 = mock(Update.class);
+    when(update1.getOldMetacard()).thenReturn(metacards.get(0));
```

Comment:
ℹ️  Added mocks regarding `getOldMetacard` since we are now using that instead of querying for the metacards in `Historian`